e2e/cypress/tests/integration/enterprise/guest_accounts/guest_identification_spec.ts

```diff
@@ -41,7 +41,7 @@ describe('Guest Accounts', () => {
         });
 
         // # Log in as a team admin.
-        cy.apiAdminLogin().then(({user}) => {
```

Comment:
I could be wrong but on quick check, this seemed to be correct and he one to fix is the typing, isn't it? cc @mvitale1989 @furqanmlk